### PR TITLE
RD-500 Upgrade uptime GCP function to Node 14

### PR DESCRIPTION
**Pull request by Gelio**
This PR upgrades the `uptime` GCP function to use Node 14. The PR is very similar to https://github.com/cloudify-cosmo/cloudify-usage/pull/9, it uses the same implementation (see https://github.com/cloudify-cosmo/cloudify-usage/pull/9#discussion_r644046673 for reasons why the code is copied, and not reused, TL;DR: it would increase the complexity by a lot)

Only the 34c005e8cb028839007efa902ffae0eaf8072bf2 commit contains functional changes (or at least I hope so) :smile: 

Test results (all green before and after the library change):
image

### CODE REVIEW

File: functions/usage/index.test.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ describe('getGeoLocationInfo', () => {
     });
 
     it('should report an unknown problem for some problematic IP', () => {
-        return expect(() =>
+        return expect(
```
Comment (Gelio) [inline functions/usage/index.test.js:22]: A minor fix, both versions pass, but I believe this is more correct :smile:

File: functions/uptime/index.js [hunk 2]
```diff
@@ -124,57 +124,39 @@ function deleteData() {
     });
 }
 
+/**
+ * @param {string} userIP
+ */
 function getGeoLocationInfo(userIP) {
```
Comment (Gelio) [inline functions/uptime/index.js:130]: The implementation is the same as for the `usage` function, just without adding the `org` to the returned object

**Review by qooban — APPROVED** [review]
Looks good 👍🏻

**Comment by Gelio** [comment]
Great, I'll merge this PR then and let @YoniItzhak test the updated functions :slightly_smiling_face:

**Pull request merged**